android/app/src/main/assets/capacitor.config.json

```diff
@@ -9,5 +9,8 @@
 			"launchAutoHide": false,
 			"showSpinner": false
 		}
+	},
+	"server": {
+		"androidScheme": "http"
```

Comment:
Das war kein wirklicher "hack", sondern eine Einstellung, die beim Update von v5 zu v6 für uns notwendig ist. 
Bei meinen ersten Tests hatte ich das nicht auf "http" gestellt und der neue default war somit "https"
Auszug aus der [Doku](https://capacitorjs.com/docs/updating/6-0#update-androidscheme):

_**Update androidScheme**
In Capacitor 6, https is the default setting for androidScheme for existing apps to better enable Capacitor applications to make use of the system [Autofill feature](https://capacitorjs.com/docs/guides/autofill-credentials)._

_Changing the scheme is the equivalent to shipping your application on a different domain, which means any data stored in in cookies, localstorage, etc would no longer be accessible. To avoid data loss as a result of this change, if you didn't have a androidScheme entry in your Capacitor config file already set to https, you should set the scheme to http._

Reply:
Verstanden. Ich meine mich aber zu erinnern, dass es die Regel gibt, dass die Apps nur mit https kommunizieren dürfen. Meine Bedenken wären jetzt, dass die Prüfung von Google dieses Setting erkennt und die App deswegen ablehnt (ist aber nur Halbwissen).